composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "5.0.x",
+        "illuminate/support": "5.x",
```

Comment:
I think being more explicit here would be in order so:
```
">=5.0.x, <=5.1.x",
```
won't cause problems when 5.2 will be released. However lets first wait for 5.1 to be released since that can bring more changes that should be fixed ;)